raiden/network/transport/matrix/transport.py

```diff
@@ -941,7 +941,9 @@ def _process_call_messages(self, call_messages: List[ReceivedCallMessage]) -> No
             call_messages: List of signalling messages
         """
 
-        assert self._web_rtc_manager is not None, "must be set"
+        if self._stop_event.is_set():
+            return
+
```

Comment:
Hm, why is this necessary? Isn't `_handle_messages` going to return early in this case?
Also, why did you remove the assert?

Reply:
the assert needs to stay. I explained in the PR. This assertion error was hit in a test

Reply:
but actually good point how this is possible 

I added the CI failure above